src/modules/Staff/Controller/Admin.php

```diff
@@ -72,8 +72,11 @@ public function get_login(\Box_App $app)
         $service = $this->di['mod_service']('staff');
         $count = $service->getAdminsCount();
         $create = (0 == $count);
-
-        return $app->render('mod_staff_login', ['create_admin' => $create]);
+        if($this->di['auth']->isAdminLoggedIn()){
+            return $app->redirect('');
+        }else{
```

Comment:
A better CS could be to avoid using else in a function which returns on both conditions.